MVC/controllers/commentsController.js

```diff
@@ -0,0 +1,8 @@
+const {eliminateCommentById} = require('../models/commentsModel');
+
+exports.deleteCommentById = (req, res, next) => {
+    const {comment_id} = req.params;
+    eliminateCommentById(comment_id).then((comment) => {
+        res.status(204).send({comment});
+    }).catch(next);
+}
```

Comment:
We don't need to send a response body here as there is no data - we can just send a status and .send()